### DISCUSSION TITLE
Pleroma Compatibility

### DISCUSSION THREAD

**Discussion by antferri** [elk-zone/elk#2445] [General]
Hello, has anyone thought of making it compatible with other realities in the fediverse, like Pleroma for example?
Thanks and good job.

**Comment by Dehelssey** [comment]
@Cl0v1s is currently making Elk compatible with Akkoma for our bdx.town instance → https://github.com/BDX-town/elk

**Comment by ayo-run** [comment]
Thank you for your inquiry!

We have some fixes specifically for compatibility with other fediverse flavors, and I see some PRs that get merged to address user problems with servers that are not Mastodon-based. Please feel free to contribute or open specific issues you find as these are all very welcome.

However, we are explicit in our focus on making Elk work first with the Mastodons API to provide focus in the maintenance.
